packages/core/src/storybook/stand-alone-documentaion/testing/best-practices.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Testing/Best Practices" />
+
+# Testing Vibe Components
+
+This guide outlines the best practices for testing components in the Vibe design system. We follow specific patterns to ensure our tests are maintainable, reliable, and consistent across the codebase.
+
+## Core Testing Philosophy
```

Comment:
While this is accurate, we HAVE to mention React Testing Library's [guiding principles](https://testing-library.com/docs/guiding-principles/), and also that we recommend using [RTL's best practices for queries](https://testing-library.com/docs/queries/about/#priority)

Reply:
```
Our testing strategy is built on two fundamental principles, inspired by the [guiding principles](https://testing-library.com/docs/guiding-principles/) and [query priorities](https://testing-library.com/docs/queries/about/#priority) outlined by Testing Library.
```
how does this sound?
notice also that there is a reference to these docs already in the bottom of the doc

Reply:
Ah missed that, it should be one of the first things in my opinion, since a lot of devs should be familiar with the concepts. 

I think your suggestion is great :D